Dockerfile

```diff
@@ -56,15 +56,15 @@ RUN cp yices*/bin/yices-smt2 rootfs/usr/local/bin
 # Install cvc4 1.8
 RUN curl -L https://github.com/CVC4/CVC4/releases/download/1.8/cvc4-1.8-x86_64-linux-opt --output rootfs/usr/local/bin/cvc4
 
-# Install Mathsat 5.6.3
-RUN curl -L https://mathsat.fbk.eu/download.php?file=mathsat-5.6.3-linux-x86_64.tar.gz | tar xz
-RUN cp mathsat-5.6.3-linux-x86_64/bin/mathsat rootfs/usr/local/bin
+# Install Mathsat 5.6.3 - Uncomment if you are in compliance with Mathsat's license.
+# RUN curl -L https://mathsat.fbk.eu/download.php?file=mathsat-5.6.3-linux-x86_64.tar.gz | tar xz
+# RUN cp mathsat-5.6.3-linux-x86_64/bin/mathsat rootfs/usr/local/bin
 
 # Set executable and run tests
 RUN chmod +x rootfs/usr/local/bin/*
 COPY --from=build /cryptol/rootfs /
 ENV PATH=/solvers/rootfs/usr/local/bin:$PATH
-RUN ! $(cryptol -c ":s prover=abc" | tail -n +2 | grep -q .) \
+RUN    ! $(cryptol -c ":s prover=abc" | tail -n +2 | grep -q .) \
     && ! $(cryptol -c ":s prover=mathsat" | tail -n +2 | grep -q .) \
```

Comment:
Oh, I suspect this line might cause trouble if MathSAT isn't installed.

Reply:
Sorry about that. Fixed.